iam/api-client/src/main/java/iam/snippets/ListServiceAccounts.java

```diff
@@ -16,64 +16,27 @@
 package iam.snippets;
 
 // [START iam_list_service_accounts]
-import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
-import com.google.api.client.json.gson.GsonFactory;
-import com.google.api.services.iam.v1.Iam;
-import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.ListServiceAccountsResponse;
-import com.google.api.services.iam.v1.model.ServiceAccount;
-import com.google.auth.http.HttpCredentialsAdapter;
-import com.google.auth.oauth2.GoogleCredentials;
+
+import com.google.cloud.iam.admin.v1.IAMClient;
 import java.io.IOException;
-import java.security.GeneralSecurityException;
-import java.util.Collections;
-import java.util.List;
 
 public class ListServiceAccounts {
 
   // Lists all service accounts for the current project.
   public static void listServiceAccounts(String projectId) {
     // String projectId = "my-project-id"
 
-    Iam service = null;
-    try {
-      service = initService();
-    } catch (IOException | GeneralSecurityException e) {
-      System.out.println("Unable to initialize service: \n" + e.toString());
-      return;
-    }
-
-    try {
-      ListServiceAccountsResponse response =
-          service.projects().serviceAccounts().list("projects/" + projectId).execute();
-      List<ServiceAccount> serviceAccounts = response.getAccounts();
-
-      for (ServiceAccount account : serviceAccounts) {
-        System.out.println("Name: " + account.getName());
-        System.out.println("Display Name: " + account.getDisplayName());
-        System.out.println("Email: " + account.getEmail());
-        System.out.println();
-      }
-    } catch (IOException e) {
-      System.out.println("Unable to list service accounts: \n" + e.toString());
+    try (IAMClient iamClient = IAMClient.create()) {
+      iamClient.listServiceAccounts("projects/" + projectId).getPage()
+              .streamValues().forEach(account -> {
```

Comment:
Can this be achieved without using getPage()? 
If the API provides implicit pagination capabilities, then we should resort to converting this to a simple for loop instead of streaming.